<launch>

<node pkg="moveit_ros_planning" type="kinematics_reachability" name="arm_workspace_tests" output="screen" >
 <param name="cache_filename" value="$(env ROS_LOG_DIR)/kinematics_workspace.cache"/>
 <param name="group" value="arm"/>
 <param name="frame_id" value="arm_base_link"/>
 <param name="kinematics_solver_timeout" value="0.1"/>
 <param name="max_fk_points" value="5000" />
</node>

</launch>
